azure-pipelines-pr.yml

```diff
@@ -0,0 +1,90 @@
+variables:
+  # Cannot use key:value syntax in root defined variables
+  - name: _TeamName
```

Comment:
Is this used anywhere?  I thought it was a requirement only for signing, so it could probably be removed.